resources/roles_edit.html

```diff
@@ -137,4 +139,10 @@
     </table>
     <input type="submit" name="submit" value="Save" />
   </form>
+
+  <form class="snooze" method="POST" action="/snooze">
+    <input type="submit" id="snooze" value="Snooze" />
+    <input name="duration" class="snooze-duration" type="number" value=5000></input>
+    <p class="snooze-desc example optional">(Snooze duration in ms)</p>
+  </form>
 </div>
```

Comment:
I think the roles page is now a little cluttered. What if we completely hid the "Snoozed until..." label/input if there is no snooze set. If there *is* a snooze set, just display a bolded, possibly red line of text underneath the Overview/Roles tabs (e.g. "**This role has been snoozed until 2015-04-21 5:00PM UTC**").